### ISSUE TITLE
Finetune model on Pascal VOC

### ISSUE THREAD

**Issue by guoyanming** [BVLC/caffe#3193]
Anyone know how to finetune the models (e.g. VGG) on Pascal VOC 2012 dataset? Since one image may contain multiple labels, I think it needs to modify the loss function, but I do not know how to do it

**Comment by ronghanghu** [comment]
Please do not post usage, installation, or modeling questions, or other requests for help to Issues.
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
